### ISSUE TITLE
cmd/compile no longer supports -R or -P

### ISSUE THREAD

**Issue by mdempsky** [rsc/toolstash#3]
In golang/go@49b2dd583beffaf983910f24c5e2d02a31d0c5b4, I removed the -R and -P flags (among others) because they don't do anything since converting to SSA.  toolstash -cmp should stop trying to use them.

**Comment by mdempsky** [comment]
@josharian I think the case should end with:

```
cmd = append([]string{cmd[0], "-v", "-m=2"}, cmd[1:]...)
log.Printf("compiler output differs, only with optimizers enabled")
```

?

**Comment by rsc** [comment]
Thanks for the help with toolstash. Now that there are others interested in working on it and reviewing it, it's probably time to move it to golang.org/x/tools/cmd. Anyone interested?

**Comment by josharian** [comment]
@mdempsky yes, thanks. Will fix.

@rsc yes! compilebench, too. I'll send a CL later. There also appear to be people using grind; I'm not going to volunteer for that one, but someone else might want to.

**Comment by josharian** [comment]
Bad commit fixed in 7508e1d.

https://go-review.googlesource.com/32681
https://go-review.googlesource.com/32682

If/when those go in, I'll update this repo to contain a README and a doc.go (with a non-main package, for godoc.org), each pointing people to the new home.

Then I'll do compilebench.

**Comment by josharian** [comment]
Move completed.
